src/runtime/classmanager.cs

```diff
@@ -470,6 +470,19 @@ private static ClassInfo GetClassInfo(Type type)
 
                 ob = new MethodObject(type, name, mlist);
                 ci.members[name] = ob;
+                if (mlist.Any(OperatorMethod.IsOperatorMethod))
+                {
+                    string pyName = OperatorMethod.GetPyMethodName(name);
+                    string pyNameReverse = OperatorMethod.ReversePyMethodName(pyName);
+                    MethodInfo[] forwardMethods, reverseMethods;
+                    OperatorMethod.FilterMethods(mlist, out forwardMethods, out reverseMethods);
```

Comment:
NIT: you can do `out var forwardMethods`. No need to declare the variables separately in the above line

Reply:
Thanks for the suggestions, I included this small change in a related PR: https://github.com/pythonnet/pythonnet/pull/1347/